src/server/pfs/fuse/loopback.go

```diff
@@ -519,25 +520,21 @@ func newLoopbackRoot(root, target string, c *client.APIClient, opts *Options) (*
 	return n, nil
 }
 
-func (n *loopbackNode) downloadRepos() (retErr error) {
-	if n.getFileState("") != none {
-		return nil
-	}
+func (n *loopbackNode) mkdirMountNames() (retErr error) {
 	defer func() {
 		if retErr == nil {
 			n.setFileState("", meta)
 		}
 	}()
-	ris, err := n.c().ListRepo()
-	if err != nil {
-		return err
-	}
-	ro := n.root().repoOpts
-	for _, ri := range ris {
-		if len(ro) > 0 && ro[ri.Repo.Name] == nil {
-			continue
-		}
-		p := n.repoPath(ri)
+	ros := n.root().repoOpts
+	// we only mount explicitly named repos now
+	for name := range ros {
+		p := n.namePath(name)
+		fmt.Printf("> MKDIR %s, %s\n", name, p)
```

Comment:
I'm not sure if this is leftover println debugging or if we want to keep these in, but if it's the latter, it might make sense to use logrus here, and log these at DEBUG? WDYT?

Reply:
removed